app/styles/columns.less

```diff
@@ -44,8 +44,8 @@
   }
 
   header {
-    border-left: 1px solid lighten(@upf-gray, 15%);
-    border-right: 1px solid lighten(@upf-gray, 15%);
+    border-left: 1px solid darken(@upf-gray-light, 10%);
+    border-right: 1px solid darken(@upf-gray-light, 10%);
```

Comment:
to check with simon, but i think we will just put a `border-color` variable in oss with a fixed hex code instead of doing darken of light of dark of light 10% :p